app/code/community/Inviqa/SymfonyContainer/Model/Observer.php

```diff
@@ -12,9 +12,15 @@ class Inviqa_SymfonyContainer_Model_Observer
     public function onCacheRefresh(Varien_Event_Observer $event)
     {
         if (ConfigurationBuilder::MODEL_ALIAS === $event->getType()) {
-            $filePath = Mage::getBaseDir('cache') . '/' . ConfigurationBuilder::CACHED_CONTAINER;
-            if (file_exists($filePath)) {
-                unlink($filePath);
+            $containerFilePath = Mage::getBaseDir('cache') . '/' . ConfigurationBuilder::CACHED_CONTAINER;
+            $metaFilePath = Mage::getBaseDir('cache') . '/' . ConfigurationBuilder::CACHED_CONTAINER . '.meta';
+
+            if (file_exists($containerFilePath)) {
```

Comment:
Maybe one more for discussion rather than a change suggestion:

Would probably extract these as there's clear duplication. On the other hand this as an extracted method is really not what I would expect to find in this class, and it is possible that the act of deleting the file is not meaningful enough in the context to deserve its own abstraction.